Breeding-Insight/bi-web
# [BI-1088] Germplasm List Export File

# Description
**Story:** [BI-1088 - Germplasm List Export File](https://breedinginsight.atlassian.net/browse/BI-1088)

- Enabled export of Germplasm Lists as xlsx file
- Modified GermplasmListsTable.vue to call the download endpoint on clicking the Download link in a row.

# Dependencies
[bi-api/BI-1088](https://github.com/Breeding-Insight/bi-api/pull/143)

# Testing
- Import multiple germplasm lists into a program.
- Go to germplasm lists table, click on download for a row
- File should download with name of Germplasm List_Date Created
- File should include only germplasm in that list ordered by entry number
- Germplasm should be in sheet called "Germplasm Import"
- File should have columns outlined in acceptance criteria with data that matches the germplasm in the list

# Checklist:

- [n/a] I have commented my code, particularly in hard-to-understand areas
- [X] I have run TAF:  https://github.com/Breeding-Insight/taf/actions/runs/1807543368
